### PR TITLE
Make node_integration_tests green on buildbot.

### LINKED ISSUE #4139: Test failure: node_integration_tests::test_exr
```____________________________ GolemNodeTest.test_exr ____________________________
self = <scripts.node_integration_tests.tests.test_golem.GolemNodeTest testMethod=test_exr>
    @unittest.skipIf(PROTOCOL_CONST.ID <= '29', "Known issue in 0.18.x")
    def test_exr(self):
        exit_code = self._run_test('golem.exr')
>       self.assertEqual(exit_code, 0)
E       AssertionError: 1 != 0
```

Seems to be caused by:

```
Task dict: {'type': 'Blender', 'name': 'test task', 'timeout': '0:10:00', 'subtask_timeout': '0:09:50', 'subtasks_count': 1, 'bid': 1.0, 'resources': ['/home/buildbot-worker/worker/test_node_integration/build/scripts/node_integration_tests/tasks/test_task_1/puchacz_canopy.jpg', '/home/buildbot-worker/worker/test_node_integration/build/scripts/node_integration_tests/tasks/test_task_1/wlochaty3.blend'], 
'options': {'output_path': '/tmp/golem-integration-test-output-v9ylx8cj', 'format': 'PNG', 'resolution': [320, 240]}}
```

Looks like `options.format = 'PNG'`

[source build log](https://buildbot.golem.network/buildbot/#/builders/15/builds/635/steps/4/logs/stdio)

### PR DESCRIPTION
**Pull request by maaktweluit**
Resolves #4139

proof: https://buildbot.golem.network/buildbot/#/builders/15/builds/643

### CODE REVIEW

File: scripts/node_integration_tests/playbooks/test_config_base.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def make_node_config_from_env(role: str, counter: int) -> NodeConfig:
 
 
 class TestConfigBase:
-    def __init__(self) -> None:
+    def __init__(self, task_settings: str = 'default') -> None:
```
Comment (etam) [inline scripts/node_integration_tests/playbooks/test_config_base.py]: I'd change that to
```python
    def __init__(self, *, task_settings: str = 'default') -> None:
```
`*` here forces using keyword arguments for setting `task_settings`, which would make it more readable and reasonable at call site.

**Review by etam — CHANGES_REQUESTED** [review]

**Review by shadeofblue — APPROVED** [review]
thank you

File: scripts/node_integration_tests/playbooks/test_config_base.py (hunk 1 above)
Comment (maaktweluit) [reply scripts/node_integration_tests/playbooks/test_config_base.py]: Good one! fixed :D

**Review by etam — APPROVED** [review]
ok

**Pull request merged**